### ISSUE TITLE
Is this project dead? 

### ISSUE THREAD

**Issue by gayanpathirage** [zachsoft/Ionic-Material#180]
Hi, I find it very interesting project and it should be the future. 

However project seems abandoned since last year (no commits, pull requests pending, issues rising, etc). 

Is there any chance that any of the contributors will kick-start this project in near future? 
Or is there a better fork which continue to support this?

**Comment by Hollerweger** [comment]
I think too that the project is very interesting and should be further developed.
Hopefully someone will continue working actively on this project.

**Comment by MT--** [comment]
I would gladly contribute my time to keep this project going. If someone could post "help wanted" on tasks or issues that need work...

**Comment by barneydmedia** [comment]
I asked about it in a Slack room, and supposedly the maintainer isn't interested in continuing to work on the project. At this point, it's probably best for someone to fork the repo and keep things going there. I love this project, and I want to implement it in the app I'm working on, but I don't have the knowledge or time to be a sole maintainer.

**Comment by MT--** [comment]
I agree with @barneydmedia. I want to implement this in a production app, but I think that it would be unwise to do so in its pre-release state. That being said, I will probably try it out to see what issues would arise. I am willing to contribute some time, but I do not have enough time to lead the charge.

**Comment by MT--** [comment]
I don't know why this didn't occur to me before:

#### Angular 2 and Ionic will be out soon. Let's fork this and start the conversion.

**Comment by Hollerweger** [comment]
Ionic 2 Beta with Angular 2 beta was released yesterday and they also
support Material Design a litte bit.
Here is the announcement:
http://blog.ionic.io/announcing-ionic-framework-2-beta/

MT-- <notifications@github.com> schrieb am Do., 11. Feb. 2016 um 14:50 Uhr:

**Comment by MT--** [comment]
Nice! I knew it was soon. **Since they are now in beta, should we start?** 

I will be doing an Ionic 2 app very soon; I am currently in the middle of prototyping a new app in Ionic 1.7.14, and I don't want to convert in the middle of Phase 1. I may convert after the prototype is  approved. I'm not very active on Github, but I have been using Ionic for quite some time now. I will try to fork this and start conversion at some point soon.

**Comment by mackelito** [comment]
So based on this issue: https://github.com/zachsoft/Ionic-Material/issues/104
Is https://github.com/IonicMaterialDesign/ionic-material-starter the new repo to use?

**Comment by NoellyB76** [comment]
Yes, dead...BUT... It lives on with all your dreams and expectations, at...

https://github.com/IonicMaterialDesign

Enjoy!

I think too that the project is very interesting and should be further
developed.
Hopefully someone will continue working actively on this project.

—
Reply to this email directly or view it on GitHub
<https://github.com/zachsoft/Ionic-Material/issues/180#issuecomment-181393720>
.

**Comment by jasonhodges** [comment]
@MT-- @barneydmedia I would like to join you guys in converting it and keeping the project alive. I am finishing up a production app in Ionic 1 but will start moving it over to 2 soon after go live. (too close now)
Please let me know how I can get involved

**Comment by MT--** [comment]
I have used *angular-material* in production many times, so I imagine my fork of this going in that direction.

I would like the Ionic/Ionic-Material Starter to be a separate repo, so that we can work on Ionic-Material-2 *then* build a starter kit.

**Comment by MT--** [comment]
http://doc.ionicmaterialdesign.com/

Looks like that is pretty recent? Hmmm....

**Comment by delta98** [comment]
Hey everyone,

As the developer and maintainer at https://github.com/IonicMaterialDesign it isn't a direct fork, in fact it uses MDL directly. If you are able to use Ionic V2 I highly recommend it as that will provide a better upgrade path. I'm really only supporting backwards compatibility and support anyone wishing to help maintain the project.

**Comment by MT--** [comment]
@delta98, awesome! Thanks for the info; I might buy it just to check it out. I know how much work went into that, and I am glad to know that it uses MDL directly. After the current project I'm working on sells or fails (I won't know until the end of April), I plan to get started on "Ionic Material 2" or something of that nature.

**Comment by christoga** [comment]
:+1: for the guys who want the project to keep developed!

**Comment by Arth95** [comment]
Whould be great if someone will maintain this project or similar one ;)

**Comment by tolyo** [comment]
Is anyone willing to maintain it?

**Comment by yeshwanthdornala** [comment]
i'm ready to maintain it.

**Comment by tolyo** [comment]
Sent invite for `https://github.com/IonicMaterial`. If you have a fork with patches in it, please init a new repo.

**Comment by yeshwanthdornala** [comment]
thanks for the invitation and i have forked present Ionic-material, shall i
cloned the present ionic-material into ionicMaterial

**Comment by tolyo** [comment]
Yes. This way anyone abandons it, the org can live with the project.

**Comment by christoga** [comment]
Big obs for all the people who want to maintain this project

**Comment by mariohmol** [comment]
Does anyone used it in production? as is using angular and ionic... i cant see why this is not for production
